transport/src/test/java/io/netty/channel/pool/FixedChannelPoolTest.java

```diff
@@ -310,6 +310,7 @@ public void run() {
             pool.release(channel).syncUninterruptibly();
             fail();
         } catch (IllegalStateException e) {
+            assertEquals("FixedChannelPool was closed", e.getMessage());
```

Comment:
@lutovich IMHO we should remove this assert as the message may change again in the future and people should really not depend on it.